vampire_survivors/src/highscore.py

```
@@ -0,0 +1,23 @@
+import pygame
```

Ruff (0.12.2)

`pygame` imported but unused

Remove unused import: `pygame`

(F401)